test/art.js

```diff
@@ -1,6 +1,8 @@
 "use strict";
 
 var clc = require("../");
+var noColor = require("../no-color");
+noColor.disable();
```

Comment:
Due to design of a test runner that's used. It'll affect all test files.

I think it'll be more bulletproof to follow with something as:

```javascript
module.exports = function (t, a) {
  noColor.disable();
  try {
    // .... tests
  } finally {
    noColors.auto();
  }
};
```